package.json

```diff
@@ -65,7 +65,8 @@
     "toad-scheduler": "^2.2.0",
     "tslib": "^2.3.0",
     "tslog": "^3.3.3",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "@aws-sdk/client-redshift-data": "^3.405.0"
```

Comment:
I may suggest you put the `@aws-sdk/client-redshift-data` package in the `extension-driver-redshift` package, then the `@aws-sdk/client-redshift-data` package will only installed when user add the `extension-driver-redshift`

You could refer to the `package.json`  of `extension-driver-clubhouse` and `extension-driver-snowflake` packages.